addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -145,7 +144,7 @@ def _get_financial_account_vals(self, partner_bank):
 
         return vals
 
-    def _get_invoice_payment_means_vals_list(self, invoice):
```

Comment:
Hmm, that's the naming thing I was talking about ... Since it is either an invoice, credit note or debit note. I find it a bit confusing when it only says `invoice` then. My rationale was that when there is no prefix, it is on the top document level, and when there is a prefix, e.g. `line_`, it is on that level.

Reply:
@dylankiss Yes I understand but the concern is more when we will implement it for sale orders, we need to distinguish which method is account.move specific or not. So yes, the `invoice` term could be confusing but it's just a way to say "account_move_having_move_type_different_from_entry" :laughing:

Reply:
@smetl Wow, that's the perfect name: `_get_account_move_having_move_type_different_from_entry_payment_means_vals_list`. I'll go with that 😂.

No, fine for me. I'll change it back to include the `invoice` word 😉